lte/protos/smfupfif.proto

```diff
@@ -0,0 +1,235 @@
+syntax = "proto3";

```

Comment:
Please remove protobuf related changes from this patch. Lets just add GTP APP in this patch.

Reply:
Yes. But in a UT test case, included the below message structures:

from lte.protos.smfupfif_pb2 import (
    SetGroupPDR,
    SetGroupFAR,
    PDI,
    LocalFTeid,
    ApplyAction,
    FwdParam,
    OuterHeaderCreation,
    SessionSet,
)
We are using these structures to passing parameters.

Reply:
ok, let me think about this a bit

Reply:
Now I have modified the definition of install/delete GTP flows as a specific arguments like ue_ip addr, incoming_TEID, outgoing_TEID, EnodeB_ip addr, precedence, IMSI/SessionID for support 4G and 5G.
def _add_gtp_tunnel_flows(self, precedence:int, i_teid:int = None,
                              o_teid:int = None, ue_ip_adr:str = None,
                              enodeb_ip_addr:str = None, sid:int = None):